frontend/src/components/FifthExamComponent/FifthExamFormComponent.tsx

```diff
@@ -1,32 +1,62 @@
-import React, { useState } from 'react';
```

Comment:
Ich hab nicht die Zeit und Geduld um mich hier einzudenken. Aber ich glaub dir dass es passt. 

Ich persönlich finde 500 Zeilen für eine Datei wesentlich zu lang. Aber von mir aus kann das so bleiben

Reply:
wunderbar.
ist nicht der schönste code, mit mehr Zeit kann mand as sicherlich auch in kleinere Datein aufteilen und alles etwas schöner machen...aber man kennt ja das Problem :/